EcosiaTests/SnapshotTests/SnapshotTestHelper.swift

```diff
@@ -107,38 +101,35 @@ final class SnapshotTestHelper {
     /// This method applies a specified theme and updates the global theme settings through a theme manager.
     ///
     /// - Parameters:
-    ///   - theme: The UIUserInterfaceStyle to set, e.g., .light or .dark.
-    ///   - suffix: The ThemeConfiguration.Theme that specifies additional theme details, typically used for naming or logging.
-    ///   - themeManager: The ThemeManager responsible for applying theme changes across the app.
+    ///   - theme: The `UIUserInterfaceStyle` to set, e.g., `.light` or `.dark`.
+    ///   - suffix: The `ThemeConfiguration.Theme` that specifies additional theme details, typically used for naming or logging.
+    ///   - themeManager: The `ThemeManager` responsible for applying theme changes across the app.
     private static func changeThemeTo(_ theme: UIUserInterfaceStyle, suffix: ThemeConfiguration.Theme, themeManager: ThemeManager) {
         LegacyThemeManager.instance.current = suffix == .light ? LegacyNormalTheme() : LegacyDarkTheme()
         themeManager.changeCurrentTheme(suffix == .light ? .light : .dark)
     }
     
-    /// Captures snapshots of a UIViewController across multiple device configurations.
+    /// Captures snapshots of a `UIViewController` across multiple device configurations.
     ///
     /// - Parameters:
-    ///   - initializer: A closure that returns a newly initialized UIViewController.
-    ///   - devices: An array of `DeviceType` for different device configurations.
-    ///   - locales: An array of `Locale` specifying the locales for the snapshot. Default to English if not specified
+    ///   - initializer: A closure that returns a newly initialized `UIViewController`.
+    ///   - locales: An array of `Locale` specifying the locales for the snapshot. Defaults to the loaded from `snapshot_configuration.json` of each test if no other array is passed.
     ///   - wait: The time interval to delay the snapshot.
-    ///   - precision: The precision of the snapshot comparison.
+    ///   - precision: The precision of the snapshot comparison. Default to 0.99 to allow slightly different colors between CI and local runs.
     ///   - file: The file in which failures should be reported.
     ///   - testName: The name of the test.
     ///   - line: The line number to report failures.
     static func assertSnapshot(
         initializingWith initializer: @escaping () -> UIViewController,
-        devices: [DeviceType] = [.iPhone12Pro_Portrait],
-        locales: [Locale] = [Locale(identifier: "en")],
+        locales: [Locale] = LocaleRetriever.getLocales(),
```

Comment:
I guess there is no reason to keep the `locales` parameter anymore, right? We can directly use `LocaleRetriever.getLocales()` inside `performSnapshot()`